deps/Makefile

```diff
@@ -591,8 +640,11 @@ update-llvm:
 	([ -d "llvm-$(LLVM_VER)/tools/lldb" ] || exit 0;            cd llvm-$(LLVM_VER)/tools/lldb; git pull --ff-only)
 
 ## LIBUV ##
+LIBUV_GIT_URL=git://github.com/JuliaLang/libuv.git
+LIBUV_TAR_URL=https://github.com/JuliaLang/libuv/tarball/$1
+$(eval $(call git-external,libuv,LIBUV,configure,.libs/libuv.a))
```

Comment:
At least checking `deps/Makefile`, I don't think any of the other instances of `.a` are using libtool, so maybe just libuv right now.